site/content/docs/5.2/components/navbar.md

```diff
@@ -323,7 +323,7 @@ Mix and match with other components and utilities as needed.
           <a class="nav-link" href="#">Pricing</a>
         </li>
       </ul>
-      <span class="navbar-text ms-3">
+      <span class="navbar-text">
         Navbar text with an inline element
```

Comment:
We may have a problem if the inline element is  a longer text, i.e. more than one line for instance. It should not happen but we never know. 
It may be possible to prevent this by using flex-basis or flex-shrink.

Reply:
For me the actual behavior with more than one line is the one wanted when it was developed I think. The fact is that the whole content should be contained inside the navbar whatever happens. The solution we tried when we worked on implementation was a `word-break: keep-all` inside nav-links but on too much items it would go outside the navbar. 

For the proposed solutions, I'd like to discuss it a bit more since I can't see how it would work but I'm not expert with flexboxes!

Not sure to understand well the Edit issue 😞

Reply:
The actual behaviour is the desired behaviour so ok form me 🚀